packages/ckeditor5-table/tests/tablecolumnresize/tablecolumnresizeediting.js

```diff
@@ -1946,6 +1946,104 @@ describe( 'TableColumnResizeEditing', () => {
 		} );
 	} );
 
+	describe( 'Hide/show resizers', () => {
+		let model, editor, view, editorElement;
+
+		beforeEach( async () => {
+			editorElement = document.createElement( 'div' );
+			document.body.appendChild( editorElement );
+			editor = await createEditor();
+
+			model = editor.model;
+			view = editor.editing.view;
+		} );
+
+		afterEach( async () => {
+			if ( editorElement ) {
+				editorElement.remove();
+			}
+
+			if ( editor ) {
+				await editor.destroy();
+			}
+		} );
+
+		describe( 'on mousedown/mouseup', () => {
+			it( 'should not hide resizers when resizer is clicked', () => {
+				setModelData( model,
+					'<table columnWidths="100%">' +
+					'<tableRow>' +
+					'<tableCell>' +
+					'<paragraph>' +
+					'[foo]' +
+					'</paragraph>' +
+					'</tableCell>' +
+					'</tableRow>' +
+					'</table>'
+				);
+
+				const resizer = view.getDomRoot().querySelector( '.table-column-resizer' );
+
+				resizer.dispatchEvent( new MouseEvent( 'mousedown', { bubbles: true } ) );
+
+				const viewRoot = view.domConverter.mapDomToView( view.getDomRoot() );
+
+				expect( viewRoot.hasClass( 'ck-resizers-hidden' ) ).to.be.false;
```

Comment:
Grouping the commands a bit differently may improve the readability of test flow:

```suggestion
                const resizer = view.getDomRoot().querySelector( '.table-column-resizer' );
                const viewRoot = view.domConverter.mapDomToView( view.getDomRoot() );

                resizer.dispatchEvent( new MouseEvent( 'mousedown', { bubbles: true } ) );

                expect( viewRoot.hasClass( 'ck-resizers-hidden' ) ).to.be.false;
```

Reply:
Also you can access a `viewRoot` a bit simpler, using `view.document.getRoot()`.